roles/tas_single_node/templates/manifests/rekor-search-ui/rekor-search-ui.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rekor-search-ui
+  namespace: rekor-search-ui
+  labels:
+    app.kubernetes.io/component: search
+    app.kubernetes.io/instance: scaffold
+    app.kubernetes.io/managed-by: Helm
```

Comment:
I don't think `managed-by: Helm` is appropriate in this file, let's just remove it.